### PR TITLE
fist draft introR practical

### PR DESCRIPTION
**Pull request by zmcucunuba**

### CODE REVIEW

File: content/post/practical-introR.Rmd [hunk 1]
```diff
@@ -0,0 +1,697 @@
...
+
+- Heterogeneous: dataframes and lists 
+
+
+
+### Atomic vectors
+
+These are the most basic structures in R and have only one dimension (1d):
+
+- a. Double (numeric)
```
Comment (zkamvar) [inline content/post/practical-introR.Rmd]: Lists and bullet points should be separate. Remove the `-`

File: content/post/practical-introR.Rmd [hunk 2]
```diff
@@ -0,0 +1,697 @@
...
+
+
+
+# Functions
+
+A function is one of the structures that makes *R* a very powerful platform for programming. 
+
+There are varios types of functions
+
+- *Primitive or base functions*: these are the default functions in _R_ under the _base_ package. For instance, for basic arithmentic operations such as addition _sum_, substraction _*_, multiplication _*_, but also for more complex operations such as extraction of median values _median_, or even more complex like _summary_ of a variable.
```
Comment (zkamvar) [inline content/post/practical-introR.Rmd]: If you want to show the symbols, show them in `code` font:

subtraction `-`, and multiplication `*`

File: content/post/practical-introR.Rmd [hunk 3]
```diff
@@ -0,0 +1,697 @@
...
+- *Functions from packages*: these are functions created within a package. For example the function *glm* in the *stats* package.
+
+
+- *User-built functions*: these are the functions that any user creates for a customised routine. In turn, these functions could become part of a package.
+
+
+The basic components of a function are:
+
+
+- __*name*:__ this is the name that we give to our function (for example: _myfun_)
```
Comment (zkamvar) [inline content/post/practical-introR.Rmd]: This is showing the stars and bold. 

`__*name*__`: __*name*__
`*__name__*`: *__name__*
`***name***`: ***name***
`___name___`: ___name___

File: content/post/practical-introR.Rmd [hunk 4]
```diff
@@ -0,0 +1,697 @@
...
+- __*formals or arguments*:__ these are the series of elements that control how to call the funciton.
+
+- __*body*:__ this is the seires of operarions or modifications on my argumnets.
+
+- __*output*:__ This is the results after mofifying the arguments. If this output corresponds to a series of data we can extract it using the command _return_.
+
+- __*function internal environment*:__ means the specific rules and obnjects within a function. Those rules and objects will not work outside the function. 
+
+
+##### User-built function (example 1)
```
Comment (zkamvar) [inline content/post/practical-introR.Rmd]: Make this a level 4 header

File: content/post/practical-introR.Rmd [hunk 5]
```diff
@@ -0,0 +1,697 @@
...
+tail(measles_dat)
+
+```
+
+
+
+From the package  *tidyr*, the most common functions are:
+
+__gather__: makes wide data 
+__spred__ : makes long data
```
Comment (zkamvar) [inline content/post/practical-introR.Rmd]: change this to "spread"

File: content/post/practical-introR.Rmd [hunk 6]
```diff
@@ -0,0 +1,697 @@
...
+# ggplot2
+
+GGplot is an implementation of the concept of *grammar of grapphics* that has been implemmented in R by Hadley Wikcham. He explains in his ggplot2 book that "the grammar is a mapping from data to aesthesic atributes (colour, shape, size) of geomethic objects (points, lines, bars)". 
+
+The main components of a ggplot2 plot are:
+- _data frame_ 
+
+- _aesthesic mappings_ this reffers to the indications on how teh data should be mapped (x, y) to color, size, etc
+
+_geoms_ reffers to geometric objects such as points, lines, shapes
```
Comment (zkamvar) [inline content/post/practical-introR.Rmd]: Should this be included in the list?

File: content/post/practical-introR.Rmd [hunk 7]
```diff
@@ -0,0 +1,697 @@
...
+
+_facets_  for condictional plors
+
+_coodinate system_ 
+
+### Basic functions in ggplot
+
+_ggplo()_ is the core function in ggplot2. The basic argument of the function is the dataframe we want to plot _(data)_.
```
Comment (zkamvar) [inline content/post/practical-introR.Rmd]: for functions, it's preferred to use the code font: `ggplot()`

File: content/post/practical-introR.Rmd [hunk 8]
```diff
@@ -0,0 +1,697 @@
...
+
+_ggplot(data)_ then can be associated using the simbol _+_ to other type of functions the __geoms_ that will  draw a particular type of shape. Some of the most commonly used are:
+
+- _geompoints()_ : for points
```
Comment (zkamvar) [inline content/post/practical-introR.Rmd]: having code font will help with the correct function names.

**Comment by zmcucunuba** [comment]
Hi @zkamvar many thanks for the code review. I have made various changes throughout, and rendered at the end.  please let me know :)

File: content/post/practical-introR.Rmd (hunk 7 above)
Comment (zmcucunuba) [reply content/post/practical-introR.Rmd]: Got it

File: content/post/practical-introR.Rmd (hunk 5 above)
Comment (zmcucunuba) [reply content/post/practical-introR.Rmd]: Thanks

File: content/post/practical-introR.Rmd (hunk 4 above)
Comment (zmcucunuba) [reply content/post/practical-introR.Rmd]: Got it

File: content/post/practical-introR.Rmd (hunk 2 above)
Comment (zmcucunuba) [reply content/post/practical-introR.Rmd]: Got it

**Comment by zkamvar** [comment]
This is weird and frustrating. The deploy is failing and netlify's deploy logs happen to be down so I can't figure out what's wrong. I know that it doesn't deploy with the last commit, but it's not clear WHY it's not deploying

**Comment by zkamvar** [comment]
I think what I'm going to do is create a new branch here using your practical and try to re-render

**Comment by zkamvar** [comment]
I'm going to close this in favor of #65 because there are issues that I cannot resolve here.

**Pull request closed without merging**